ehealth-portal-backend/manifests/ehealth-portal-backend-deployment.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ehealth-portal-backend
+  namespace: openncp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ehealth-portal-backend
+  template:
+    metadata:
+      labels:
+        app: ehealth-portal-backend
+    spec:
+      containers:
+      - name: ehealth-portal-backend
+        image: docker.io/3ahealth/ehealth-portal-backend:7.1.0
+        ports:
```

Comment:
This should not be needed, please review and verify

Reply:
Yes, you are right! Fixed!